### PR TITLE
[SYCLomatic] Add api query mapping for some warp APIs

### PR DESCRIPTION
**Pull request by intwanghao**

### CODE REVIEW

**Review by tangjj11 — APPROVED** [review]
lgtm

File: clang/test/dpct/query_api_mapping/Runtime/test_warp.cu [hunk 1]
```diff
@@ -0,0 +1,144 @@
+// UNSUPPORTED: cuda-8.0
+// UNSUPPORTED: v8.0
```
Comment (tangjj11) [inline clang/test/dpct/query_api_mapping/Runtime/test_warp.cu]:
```suggestion
// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0
```

File: clang/examples/DPCT/Runtime/__activemask.cu [hunk 2]
```diff
@@ -0,0 +1,5 @@
+__device__ void test(unsigned int r) {
+  // Start
+  r = __activemask();
```
Comment (zhimingwang36) [inline clang/examples/DPCT/Runtime/__activemask.cu:3]: The mapping for this API is not exactly correct, we need put DPCT1086 in mapping. 
Please sync with Jiajun on the final update.

Comment (intwanghao) [reply clang/examples/DPCT/Runtime/__activemask.cu:3]: Synced with @tangjj11 the warning will be remain.

**Review by tangjj11 — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**